src/widgets/modalChildren/PatientEditModal.js

```diff
@@ -22,62 +21,53 @@ export const PatientEditModalComponent = ({
   inputConfig,
   surveySchema,
   surveyForm,
-  onOpen,
   onUpdateForm,
   nameNoteIsValid,
-}) => {
-  useEffect(() => {
-    if (surveySchema) {
-      onOpen();
-    }
-  }, []);
-
-  return (
-    <FlexRow style={{ flexDirection: 'column' }} flex={1}>
-      <FlexRow flex={1}>
-        <FormControl
-          canSave={surveySchema ? nameNoteIsValid : true}
-          isDisabled={isDisabled}
-          onSave={onSaveForm}
-          onCancel={onCancel}
-          inputConfig={inputConfig}
-          showCancelButton={false}
-          showSaveButton={false}
-        />
-        {surveySchema && surveyForm && (
-          <View style={styles.formContainer}>
-            <JSONForm
-              surveySchema={surveySchema}
-              formData={surveyForm}
-              onChange={({ formData }, validator) => {
-                onUpdateForm(formData, validator);
-              }}
-            >
-              <></>
-            </JSONForm>
-          </View>
-        )}
-      </FlexRow>
-      <FlexRow flex={0} style={{ justifyContent: 'center' }}>
-        <View style={styles.buttonsRow}>
-          <PageButton
-            onPress={onSaveForm}
-            style={styles.saveButton}
-            isDisabled={!nameNoteIsValid || isDisabled}
-            textStyle={styles.saveButtonTextStyle}
-            text={generalStrings.save}
-          />
-          <PageButton
-            onPress={onCancel}
-            style={styles.cancelButton}
-            textStyle={styles.cancelButtonTextStyle}
-            text={modalStrings.cancel}
-          />
+}) => (
```

Comment:
don't need the `onOpen` now, because it is creating the required records in `DispensingPage` callbacks